README.md

```diff
@@ -343,6 +367,23 @@ The above looks like the following in `config.xml`:
     <splash src="res/screen/ios/Default@2x~ipad~comany.png" />
 ```
 
+##### Dark Mode
+
+Since [Cordova-ios@6.1.0](https://github.com/apache/cordova-ios) it is now possible to use optionally different splashscreen images when your app is running in dark mode. You can configure these images in config.xml with the ~dark suffix (and ~light is also supported).
```

Comment:
```suggestion
Since [Cordova-ios@6.1.0](https://github.com/apache/cordova-ios) it is now possible to optionally specify different SplashScreen images to be used when your app is running in dark mode. You can specify the luminosity of SplashScreen images in config.xml using the ~dark and ~light suffixes.
```